src/templates/subscribe.html

```diff
@@ -0,0 +1,30 @@
+{% extends "base.html" %}
+{% block content %}
+<div class="container">
+    <h1>Get updated</h1>
+    <p>Get the newst Festival news right into your inbox, for free. As soon as something changes, you will be notfied.</p>
+    <div class="row">
+        <div class="col-5">
+
+            <form action="{{ url_for('success') }}" method="post">
+              <div class="form-group">
+                <label for="exampleInputEmail1">Email address*</label>
+                <input type="email" class="form-control" id="email" name="email" aria-describedby="emailHelp" required="true" placeholder="Enter email">
```

Comment:
For required input you can just add the attribute without the `="true"`. If the attribute is there it already implies that it's true.